OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -397,7 +397,7 @@ bool JammedBy(TraitPair<JamsMissiles> tp)
 
 		void ChangeSpeed(int sign = 1)
 		{
-			speed = (speed + sign * info.Acceleration.Length).Clamp(0, info.MaximumSpeed.Length);
+			speed = (speed + sign * info.Acceleration.Length).Clamp(0, info.MaximumLaunchSpeed.Length + info.MaximumSpeedBonus.Length);
```

Comment:
For clarity: My suggestion above is basically regarding this line, which implicitly defines the cruising speed of the missile as the sum of two other speeds.  I'm suggesting that we use a single `info.Speed.Length` here, and then adjust the other speeds to make it work out.